src/helperFunctions.js

```diff
@@ -11,3 +11,45 @@ export const checkErrorMessage = (error, message) => {
     notify.show(message, 'error', 3000);
   }
 };
+export const mapOrder = (arrayToSort, arrayToOrderBy, keyToSortBy) => {
```

Comment:
I think this name could be more descriptive. Something like `sortSources` maybe?.

Reply:
I thought this being a helper method that could be used somewhere else it needed to have a general name. Just in case in the future, we need to map another array against a specific order we could use the same function.

Reply:
That makes sense, but I still think the name could improve. `mapOrder` is still not very clear. `prioritiseItems` maybe?

Reply:
Alright, let me work on it.